Login gui

Anders-Bacon opened the pull request:
Frontend part of Login and Create account page.
The pages are currently routing to the welcomepage, because their correct routes dont exists yet.
The css is rensposive and works on all devices.

AndersBaconThomsen committed:
HTML & CSS for Login page

Beginning stage of html and css of the login page. It has yet to be centered

AndersBaconThomsen committed:
Merge branch 'main' into login-gui

AndersBaconThomsen committed:
Fixing merge conflict

Merge conflicts of web.php and login page and login.css

AndersBaconThomsen committed:
New Account page

new accoutn page with added css. It currently routes to the welcome page so thats a thing that needs fixing but otherwise its done

MidnightRocket reviewed (changes requested):
I think that files in "resources/users/" should be moved to "resources/account" for account related stuff. 
I also think that the url endpoint "/newAcc" should be "/signup" to better meet standards.

MidnightRocket commented on resources/views/users/newAcc.blade.php:
```diff
@@ -0,0 +1,37 @@
+        </article>
+    </div>
+
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
```
I think these script tags should be moved into the html head. And then use the `defer` attribute to defer execution until document load.

Anders-Bacon replied:
Everything should be fixed to the feedback recieved. the scripts are now in the head, with a defer attribute and every path, and pathname have been fixed as well.

AndersBaconThomsen committed:
Change of routes to match standards

Changing route newAcc to signup to meet standards

AndersBaconThomsen committed:
Script change

The scripts are now placed in the header, and have the defer attribute so that their execution waits until document load

AndersBaconThomsen committed:
Fixing small details

Fixed a duplication in routing, so that it now has the correct path for views, and a duplication of a route. And added a break link to have the button more split from the input tag in login.

MidnightRocket commented on resources/views/users/newAcc.blade.php:
What about these two files in users?
